src/components/bars/OnboardingHeader/index.tsx

```diff
@@ -133,6 +134,9 @@ const OnboardingHeader: FC<HeaderBarProps> = ({title, stepConfig, onBack, header
             type: OnboardingMachineEvents.UPDATE_SECURITY_MODEL,
             model: securityModel,
           });
+          if(securityModel === PIDSecurityModel.REMOTE_HSM) {
```

Comment:
I think this is duplicate now, is done already in storagePersistPIDSecurityModel